src/logger-configuration.ts

```diff
@@ -4,4 +4,7 @@ export interface LoggerConfiguration {
     logstashPort: number;
     writeToConsole?: boolean;
     writeFullMessageToConsole?: boolean;
+    logFilePath?: string;
```

Comment:
I see a little problem here.
If someone provides a path to a log file, it might be a prepared file that was already created.
But than, you add the suffix of the date to the log file name, so the logs will be written in the new file and not in the file expected by the client.

Maybe its better to only provide the logs directory path, and than logs will be called in some default name of our own?